### PR TITLE
Add EmptyExampleGroup cop

### PR DESCRIPTION
**Pull request by backus**

### CODE REVIEW

**Comment by backus** [comment]
@andyw8 and @agrimm could you review this? This is an implementation of #56

File: lib/rubocop/cop/rspec/empty_example_group.rb [hunk 1]
```diff
@@ -0,0 +1,76 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module RSpec
+      # Checks if test is focused.
```
Comment (andyw8) [inline lib/rubocop/cop/rspec/empty_example_group.rb]: Wrong comment?

File: lib/rubocop/cop/rspec/empty_example_group.rb [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+      #
+      # @example
```
Comment (andyw8) [inline lib/rubocop/cop/rspec/empty_example_group.rb]: Should this document the `CustomIncludeMethods` option?

File: lib/rubocop/cop/rspec/empty_example_group.rb (hunk 1 above)
Comment (backus) [inline lib/rubocop/cop/rspec/empty_example_group.rb]: Ah shucks. Thanks

File: lib/rubocop/cop/rspec/empty_example_group.rb (hunk 2 above)
Comment (backus) [inline lib/rubocop/cop/rspec/empty_example_group.rb]: Probably. I need to look into how this is usually documented then

**Pull request merged**